exporter/awsxrayexporter/translator/cause_test.go

```diff
@@ -629,3 +629,78 @@ TypeError: must be str, not int`
 	assert.Equal(t, "main.py", *exceptions[0].Stack[1].Path)
 	assert.Equal(t, 14, *exceptions[0].Stack[1].Line)
 }
+
+func TestParseExceptionWithJavaScriptStacktrace(t *testing.T) {
+	exceptionType := "TypeError"
+	message := "Cannot read property 'value' of null"
+	// We ignore the exception type / message from the stacktrace
+	stacktrace := `TypeError: Cannot read property 'value' of null
+	at speedy (/home/gbusey/file.js:6:11)
```

Comment:
It looks like these are tabbed - I think JS stacktraces will use spaces, can you confirm and update the test cases?

Reply:
yeah it uses spaces, updating